<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDataSelectionDisplayFrame</class>
 <widget class="QWidget" name="FindDataSelectionDisplayFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="pqColorChooserButton" name="selectionColor">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>&lt;p&gt;Set the color to use to show selected elements&lt;/p&gt;</string>
     </property>
     <property name="text">
      <string>Selection Color</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cellLabelsButton">
     <property name="toolTip">
      <string>&lt;p&gt;Set the array to label selected cells with&lt;/p&gt;</string>
     </property>
     <property name="text">
      <string>Cell Labels</string>
     </property>
     <property name="icon">
      <iconset resource="../pqComponents.qrc">
       <normaloff>:/pqWidgets/Icons/pqCellData16.png</normaloff>:/pqWidgets/Icons/pqCellData16.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pointLabelsButton">
     <property name="toolTip">
      <string>&lt;p&gt;Set the array to label to selected points with&lt;/p&gt;</string>
     </property>
     <property name="text">
      <string>Point Labels</string>
     </property>
     <property name="icon">
      <iconset resource="../pqComponents.qrc">
       <normaloff>:/pqWidgets/Icons/pqPointData16.png</normaloff>:/pqWidgets/Icons/pqPointData16.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="showFrustumButton">
     <property name="toolTip">
      <string>&lt;p&gt;For frustum based selections, show the frustum used to select the elements&lt;/p&gt;</string>
     </property>
     <property name="text">
      <string>Show frustum</string>
     </property>
     <property name="icon">
      <iconset resource="../pqComponents.qrc">
       <normaloff>:/pqWidgets/Icons/pqFrustumSelectionCell24.png</normaloff>:/pqWidgets/Icons/pqFrustumSelectionCell24.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="showLabelPropertiesButton">
     <property name="toolTip">
      <string>Edit selection label properties</string>
     </property>
     <property name="icon">
      <iconset resource="../pqComponents.qrc">
       <normaloff>:/pqWidgets/Icons/pqAdvanced26.png</normaloff>:/pqWidgets/Icons/pqAdvanced26.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqColorChooserButton</class>
   <extends>QToolButton</extends>
   <header>pqColorChooserButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
